Makefile

```diff
@@ -78,12 +78,12 @@ manifests: controller-gen
 
 # Run go fmt against code
 fmt:
-	go fmt ./...
-
-# Run go vet against code
-vet:
```

Comment:
Is it possible to keep vet?

Reply:
golangci-lint includes govet as part of its checks (and is enabled in the config I added) so I didn't think that added much value?

Reply:
Re-added as per discussion